test/test.js

```diff
@@ -0,0 +1,34 @@
+const mockdb = require('../mockdb')
+const mockDb = new mockdb.mockDb(mockdb.availableItems)
+
+var expect = require('chai').expect;
+
+describe('itemsByLocation(...)', function () {
+    it('should return all items in a window', function () {
+        tlLocation = {
+            lat: 52,
+            lng: 12
+        };
+        lrLocation = {
+            lat: 54,
+            lng: 14
+        };
+        items = mockDb.itemsByLocation(tlLocation, lrLocation)
+        expect(items.length).to.be.equal(mockdb.availableItems.length);
+    })
+})
+
+describe('itemsByLocation(...)', function () {
+    it('should return no items in an invalid window', function () {
```

Comment:
Is this  an invalid window or a valid window without items?

Reply:
I added validation to the request handler function. This makes more sense in the long term, as the MockDd class is only useful as long as we don't have an actual database.